ui/raidboss/data/03-hw/dungeon/gubal_library_hard.js

```diff
@@ -62,7 +62,7 @@
     },
     {
       id: 'Gubal Hard Ferrofluid',
-      regex: / 1B:(\y{ObjectId}):(\y{Name}):....:....:(0030|0031):/,
+      regex: / 1B:(\y{ObjectId}):(\y{Name}):....:....:(0030|0031): /,
```

Comment:
I don't think this space at the end should be added.

Reply:
sry for that, reverted the change